wsi_deid/import_export.py

```diff
@@ -100,13 +100,28 @@ def validateDataRow(validator, row, rowNumber, df):
     return errors
 
 
+def getSchema():
+    """
+    Return a jsonschema.
+
+    :returns: an object that can be passed to the jsonschema validator.
+    """
+    # TODO: When we have a schema folder, if it exists and any items are in it
+    # that contain a single file that parses as valid json, then we need to
+    # load those files and combine them into a single schema in lieu of using
+    # this default SCHEMA_FILE_PATH
+    logger.info('Using %s (length %d) for schema' % (
```

Comment:
@marySalvi Once we have a schema folder, we'll add code here to take advantage of it.